docs/configuration.md

```diff
@@ -1523,6 +1523,15 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
+  <td><code>spark.streaming.backpressure.initialRate</code></td>
+  <td>not set</td>
+  <td>
+    Initial rate for backpressure mechanism (since 1.5). This provides maximum receiving rate of
+    receivers in the first batch when enables the backpressure mechanism, then the maximum receiving
```

Comment:
I'd say: "This is the initial maximum receiving rate at which each receiver will receive data for the first batch when the backpressure mechanism is enabled."